lagtraj/forcings/conversion/load.py

```diff
@@ -56,19 +56,15 @@ def load_definition(root_data_path, forcing_name, conversion_name):
         method=conversion_params.get("levels_method", None),
     )
 
-    conversion_nudging_momentum_definition = ConversionNudgingDefinition(
-        method=conversion_params.get("nudging_method_momentum_traj", None),
-        time=conversion_params.get("nudging_time_momentum_traj", None),
-        height=conversion_params.get("nudging_height_momentum_traj", None),
-        transition=conversion_params.get("nudging_transition_momentum_traj", None),
-    )
+    nudging_param_values = {}
 
-    conversion_nudging_scalar_definition = ConversionNudgingDefinition(
-        method=conversion_params.get("nudging_method_scalar_traj", None),
-        time=conversion_params.get("nudging_time_scalar_traj", None),
-        height=conversion_params.get("nudging_height_scalar_traj", None),
-        transition=conversion_params.get("nudging_transition_scalar_traj", None),
-    )
+    for k in NUDGING_REQUIRED_FIELDS:
+        value = conversion_params.get(k)
+        if value is None:
+            # TODO: implement default values here
```

Comment:
should we maybe raise an exception here?
```suggestion
            raise NotImplementedError(value)
```

Reply:
The exception leads to problems passing the tests. I think the issue is that we do want these to be None.